### PR TITLE
Configurable LinkedIn attributes

### PR DESCRIPTION
**Pull request by tauceti2**
- Attributes which can be get from LinkedIn are now configurable
- By default these attributes are requested: id,first-name,last-name,headline,summary,specialties,picture-url,email-address
- Added email-address attribute into the default set
- Fixed link to the documentation

### CODE REVIEW

File: modules/authlinkedin/docs/oauthlinkedin.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Using the LinkedIn authentication source with SimpleSAMLphp
 ===========================================================
 
-Remember to configure `authsources.php`, with both Consumer key and secret.
+Remember to configure `authsources.php`, with both Consumer key and secret. Optionally you can set which attributes are requested from LinkedIn, the list of available attributes can be foudnd here <https://developer.linkedin.com/docs/fields>.
```
Comment (thijskh) [inline modules/authlinkedin/docs/oauthlinkedin.md]: Typo: foudnd

File: modules/authlinkedin/lib/Auth/Source/LinkedIn.php [hunk 2]
```diff
@@ -46,6 +47,12 @@ public function __construct($info, $config) {
 			throw new Exception('LinkedIn authentication source is not properly configured: missing [secret]');
 
 		$this->secret = $config['secret'];
+
+		if (array_key_exists('attributes', $config))
+			$this->attributes = $config['attributes'];
+		else
+			// Default values if the attributes are not set in config (ref https://developer.linkedin.com/docs/fields)
+			$this->attributes = 'id,first-name,last-name,headline,summary,specialties,picture-url,email-address';
 	}
```
Comment (thijskh) [inline modules/authlinkedin/lib/Auth/Source/LinkedIn.php]: Can you please add braces to the control structure? We're following PSR-2.

**Comment by thijskh** [comment]
Thanks! Seems like a fine improvements, just had some small comments on the code.

File: modules/authlinkedin/lib/Auth/Source/LinkedIn.php (hunk 2 above)
Comment (tauceti2) [inline modules/authlinkedin/lib/Auth/Source/LinkedIn.php]: Do you want me to fix the rest of the original file in order to follow PSR-2?

Comment (thijskh) [inline modules/authlinkedin/lib/Auth/Source/LinkedIn.php]: That would be ideal, yes.

Comment (jaimeperez) [inline modules/authlinkedin/lib/Auth/Source/LinkedIn.php]: I just have one request if you fix the entire file to follow PSR-2, and it's that you do that in separate commits, so that one commit is changing the file to be PSR-2 compliant, and the other is adding the functionality that you want to merge. That way we keep a clean history and it's easier to parse the changes and evaluate the PR.

Thanks a lot for your work @tauceti2!

**Comment by thijskh** [comment]
Thanks. I've taken the liberty to squash the four commits into one and merged it. If you'd like to provide a PSR-2 updated version of the entire file that is also still very welcome.
